@here/harp-vectortile-datasource/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
```

Comment:
Please update the copyright statement to include the last modified year e.g. 2020 so it's "Copyright (C) 2017-2019 HERE Europe B.V."